build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
     testImplementation "org.springframework.security:spring-security-test"
     testImplementation "org.springframework.kafka:spring-kafka-test"
     testImplementation libraries["wiremock"]
+    testImplementation libraries["awaitility"]
```

Comment:
I used to work a lot with this test library and I think it's quite useful (even Sonar recommends adding it to test timeout scenarios like the one I added as part of this pull request).